extension/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "scripts": {
+    "generate-vapid-keys": "yarn workspace @se-2/nextjs web-push generate-vapid-keys --json"
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
Why do you need this?

Reply:
generate-vapid-keys is used to generate vapid keys that are used for notifications. (Let me know if I am answering your question correctly)

I think we can leave the packageManager field out as it is optional. Thanks!

Reply:
Yeah, I mean the packageManager field. Thanks!